packages/patternfly-4/react-core/src/components/Select/Select.d.ts

```diff
@@ -3,12 +3,15 @@ import { HTMLProps, FormEvent, ReactNode } from 'react';
 export const SelectVariant: {
   single: 'single';
   checkbox: 'checkbox';
+  typeahead: 'typeahead';
+  typeahead_multi: 'typeaheadmulti';
```

Comment:
Curious why we are not using CamelCase here?

Reply:
I can change it